bigtable-hbase-2.x-parent/bigtable-hbase-2.x-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestTruncateTable.java

```diff
@@ -38,4 +39,15 @@ protected void createTable(TableName tableName, byte[][] splits) throws IOExcept
     }
   }
 
+  /**
+   * @throws IOException
+   */
+  @Test
+  public void testTruncateTableAsync() throws IOException{
```

Comment:
Please create a new subclass of `AbstractTestTruncateTable` to test `admin.truncateTableAsync()`.